diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter6/classes/CompactInitialization.kt

```diff
@@ -81,7 +81,7 @@ class CompactInitialization(private val configRules: List<RulesConfig>) : Rule("
             }
     }
 
-    @Suppress("UnsafeCallOnNullableType", "NestedBlockDepth")
+    @Suppress("UnsafeCallOnNullableType", "NestedBlockDepth", "COMMENTED_OUT_CODE")
```

Comment:
Same here

Reply:
Because there is comment with `apply { foo(this) }`